_posts/2018-09-20-release-1.6.1.md

```diff
@@ -0,0 +1,185 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10150'>FLINK-10150</a>] -         Chained batch operators interfere with each other other
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10151'>FLINK-10151</a>] -         [State TTL] Fix false recursion call in TransformingStateTableKeyGroupPartitioner.tryAddToSource
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10154'>FLINK-10154</a>] -         Make sure we always read at least one record in KinesisConnector
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10169'>FLINK-10169</a>] -         RowtimeValidator fails with custom TimestampExtractor
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10172'>FLINK-10172</a>] -         Inconsistentcy in ExpressionParser and ExpressionDsl for order by asc/desc
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10192'>FLINK-10192</a>] -         SQL Client table visualization mode does not update correctly
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10193'>FLINK-10193</a>] -         Default RPC timeout is used when triggering savepoint via JobMasterGateway
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10204'>FLINK-10204</a>] -         StreamElementSerializer#copy broken for LatencyMarkers 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10255'>FLINK-10255</a>] -         Standby Dispatcher locks submitted JobGraphs
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10261'>FLINK-10261</a>] -         INSERT INTO does not work with ORDER BY clause
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10267'>FLINK-10267</a>] -         [State] Fix arbitrary iterator access on RocksDBMapIterator
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10269'>FLINK-10269</a>] -         Elasticsearch 6 UpdateRequest fail because of binary incompatibility
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10283'>FLINK-10283</a>] -         FileCache logs unnecessary warnings
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10293'>FLINK-10293</a>] -         RemoteStreamEnvironment does not forward port to RestClusterClient
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10314'>FLINK-10314</a>] -         Blocking calls in Execution Graph creation bring down cluster
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10328'>FLINK-10328</a>] -         Stopping the ZooKeeperSubmittedJobGraphStore should release all currently held locks
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10329'>FLINK-10329</a>] -         Fail with exception if job cannot be removed by ZooKeeperSubmittedJobGraphStore#removeJobGraph
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10330'>FLINK-10330</a>] -         Code Compilation failed in local mode
```

Comment:
We also need to update the issues here. It should now contain 3 JIRAs less.

Reply:
will do